src/stylish-text-generator/stylish-text-generator.ts

```diff
@@ -0,0 +1,298 @@
+                        <h4>Canvas Size</h4>
+                        <input type="range" class="range-input" min="300" max="1200" step="50" .value=${this.canvasWidth.toString()} @input=${(e: any) => {
+                            this.canvasWidth = parseInt(e.target.value);
+                            this.generateImage();
+                        }}>
+                        <div class="value-display">Width: ${this.canvasWidth}px</div>
+                        <input type="range" class="range-input" min="100" max="800" step="25" .value=${this.canvasHeight.toString()} @input=${(e: any) => {
+                            this.canvasHeight = parseInt(e.target.value);
+                            this.generateImage();
+                        }}>
+                        <div class="value-display">Height: ${this.canvasHeight}px</div>
+                    </div>
+                    
+                    <div class="control-group">
+                        <h4>Font</h4>
+                        <select class="font-select" .value=${this.fontFamily} @change=${(e: any) => {
+                        this.fontFamily = e.target.value;
+                        setTimeout(() => this.generateImage(), 100);
+                         }}>
+                         ${this.fontOptions.map(font => 
+                            html`<option value="${font.name}">${font.name}</option>`
+                        )}
+                        </select>
+                        <input type="range" class="range-input" min="12" max="120" .value=${this.fontSize.toString()} @input=${(e: any) => {
+                            this.fontSize = parseInt(e.target.value);
+                            this.generateImage();
+                        }}>
+                        <div class="value-display">Size: ${this.fontSize}px</div>
+                    </div>
+
+                    <div class="control-group">
+                        <h4>Colors</h4>
+                        <label>Text Color</label>
+                        <input type="color" class="color-input" .value=${this.textColor} @input=${(e: any) => {
+                            this.textColor = e.target.value;
+                            this.generateImage();
+                        }}>
+                        <label>Background</label>
+                        <input type="color" class="color-input" .value=${this.backgroundColor} @input=${(e: any) => {
+                            this.backgroundColor = e.target.value;
+                            this.generateImage();
+                        }}>
+                    </div>
+
+                    <div class="control-group">
+                        <h4>Stroke</h4>
+                        <input type="range" class="range-input" min="0" max="10" .value=${this.strokeWidth.toString()} @input=${(e: any) => {
+                            this.strokeWidth = parseInt(e.target.value);
+                            this.generateImage();
+                        }}>
```

Comment:
let's avoid arrow function in template
you may use a generic function for this maybe `onValueChange(type, value)`
type = "strokeWidth" | "backgroundColor" | ....